### ISSUE TITLE
system-information command doesn't work correctly for publish instance

### ISSUE THREAD

**Issue by YuriyShestakov** [jlentink/aem#39]
Hi Jason!

The `aem system-information` command works well on author but outputs an error when I run it for publisher. I suppose because on publisher no replication or distribution agent, and they are tried to be parsed from the response.

```
$ aem version
AEMcli
Version: 1.0.2

$ aem system-information -n local-publish
Adobe Experience manager:
- Version: 6.5.5.0
- Run mode:
	 1) s7connect
	 2) crx3
	 3) samplecontent
	 4) publish
	 5) local
	 6) crx3tar
- Up since: 2021-06-23 18:50:01

Repostitory
- Version: 1.22.3
- Repository size: 0.36 GB
- Storage location: crx-quickstart/repository/datastore
- Nodes total: 243712
- Tags total: 93
- Authorizables: 281
- Pages: 2250

Operating system:
- MacOS 10.15.7

Maintenance Tasks:
- Failed:
	 1) AuditLog Maintenance Task
	 2) Workflow Purge

- Succeeded:
	 1) Data Store Garbage Collection
	 2) Lucene Binaries Cleanup
	 3) Revision Clean Up

- Not scheduled:
	 1) Project Purge
	 2) Version Purge
	 3) Purge of ad-hoc tasks

Current health checks:
- 1 Critical:
	 1) System Maintenance

- 3 Warn:
	 1) Log Errors
	 2) Security Checks
	 3) Sling/Granite Content Access Check

Replication Agents:
- Idle:
	 1) outbox

panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 1 [running]:
github.com/jlentink/aem/internal/commands.(*commandSystemInformation).run(0x1d50940, 0xc0003cf340, 0xc0003c96c0, 0x0, 0x2)
	/Users/utopia/go/src/github.com/jlentink/aem/internal/commands/systemInformation.go:102 +0xe30
github.com/spf13/cobra.(*Command).execute(0xc0003cf340, 0xc0003c96a0, 0x2, 0x2, 0xc0003cf340, 0xc0003c96a0)
	/Users/utopia/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:842 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1d2d940, 0xc0004e7f58, 0x1, 0x1)
	/Users/utopia/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:943 +0x336
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/utopia/go/pkg/mod/github.com/spf13/cobra@v0.0.7/command.go:883
github.com/jlentink/aem/internal/commands.Execute()
	/Users/utopia/go/src/github.com/jlentink/aem/internal/commands/commands.go:48 +0x1b5
main.main()
	/Users/utopia/go/src/github.com/jlentink/aem/main.go:8 +0x25
```

**Comment by jlentink** [comment]
Hi @YuriyShestakov,

Thanks for the ticket. I'll have a look and will resolve the issue.

**Comment by jlentink** [comment]
The issue has been resolved in release in v1.0.3.
